src/puppet-padplus.ts

```diff
@@ -374,7 +374,7 @@ export class PuppetPadplus extends Puppet {
     }
     const payload = await this.manager.getFriendship(friendshipId)
     if (!payload || payload.type !== FriendshipType.Receive) {
-      throw new Error(`friendship type error.`)
+      throw new Error(`can not find friendship payload ${payload} or friendship type ${payload?.type} error.`)
```

Comment:
${payload} you can only get a string like `[Object object]`

Reply:
agree

Reply:
fix